addons/hr_recruitment_integration_monster/data/hr_contract_type_data.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <function model="hr.contract.type" name="write">
+            <value model="hr.contract.type" search="[
```

Comment:
Use external identifiers instead of searching on name (which is not working with translated terms, or if the name if manually changed).

Reply:
yup